### PR TITLE
Basically completed

### PR DESCRIPTION
**Pull request by 1340908470**
SDK with README.md

### CODE REVIEW

**Comment by 1340908470** [comment]
[Demo app using this sdk](https://bupteducn-my.sharepoint.com/:u:/g/personal/1340908470_bupt_edu_cn/EUpLpCXpjd1IgJEJ45UBUT0BnP-vZqdICtJ9hl1bGuY2XQ?e=592rfJ)

**Comment by 1340908470** [comment]
@hsluoyz

File: .gitignore [hunk 1]
```diff
@@ -1,85 +1 @@
...
-# Version control
-vcs.xml
-
-# lint
-lint/intermediates/
-lint/generated/
-lint/outputs/
-lint/tmp/
-# lint/reports/
+/build
```
Comment (hsluoyz) [inline .gitignore]: why delete?

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Comment by hsluoyz** [comment]
@sh1luo @tangyang9464 @Kininaru @WindSpiritSR plz review

**Pull request merged**